main.go

```diff
@@ -78,13 +78,20 @@ func main() {
 	app.UseJWTMiddleware(service, jwt.New(publicKey, nil, app.NewJWTSecurity()))
 
 	// Mount "login" controller
-	c := NewLoginController(service)
-	app.MountLoginController(service, c)
+	loginCtrl := NewLoginController(service)
+	app.MountLoginController(service, loginCtrl)
 	// Mount "version" controller
-	c2 := NewVersionController(service)
-	app.MountVersionController(service, c2)
+	versionCtrl := NewVersionController(service)
+	app.MountVersionController(service, versionCtrl)
 
 	// Mount "workitem" controller
+	workitemCtrl := NewWorkitemController(service, wiRepo, ts)
+	app.MountWorkitemController(service, workitemCtrl)
+
+	// Mount "workitemtype" controller
+	workitemtypeCtrl := NewWorkitemtypeController(service, witRepo, ts)
+	app.MountWorkitemtypeController(service, workitemtypeCtrl)
+
 	c3 := NewWorkitemController(service, wiRepo, ts)
 	app.MountWorkitemController(service, c3)
 
```

Comment:
@tsmaeder : Sorry to make a comment on Github and so late, but I want to understand that we already have mounted controllers for WI and WIT then do we need to do it again ? using c3 and c4 , Can I remove last two parts. I think it is redundancy. Let me know if i am missing anything.